packages/gatsby-cli/src/reporter/index.js

```diff
@@ -65,8 +55,8 @@ module.exports = Object.assign(reporter, {
       error = message
       message = error.message
     }
-    base.error.call(this, message)
-    if (error) console.log(errorFormatter.render(error))
+    // base.error.call(this, message)
+    if (error) this.log(errorFormatter.render(error))
```

Comment:
While we're here--we should probably fix this to use the correct code (e.g. .error). This was something that maybe is impacting cloud re: stripping ANSI color codes.

Reply:
`errorFormatter` should be set with no color when the function is being set:
https://github.com/gatsbyjs/gatsby/pull/13089/files#diff-8a16e03a18503e23944418632510f11bR34

I'll still need to check if it also strips already added colors.